Client/Frontend/Browser/MetadataParserHelper.swift

```diff
@@ -84,6 +82,6 @@ class MediaImageLoader: TabEventHandler {
     }
 
     fileprivate func cache(image: UIImage, forURL url: URL) {
-        SDWebImageManager.shared().saveImage(toCache: image, for: url)
+        SDImageCache.shared.storeImage(toMemory: image, forKey: url.absoluteString)
```

Comment:
I think `SDWebImageManager.shared().saveImage(toCache` was saving this image to the disk if I'm not mistaken